From: Michal Kubecek <mkubecek@suse.cz>
Date: Mon, 10 Oct 2016 01:30:00 +0200
Subject: kabi: hide new member recursion_counter in struct sk_buff
Patch-mainline: Never, kabi workaround
References: CVE-2016-7039 bsc#1001486

The fix for bsc#1001486 inserts a new bitfield recursion_counter into an
existing hole in struct sk_buff. Hide this change from kABI checker.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 include/linux/netdevice.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index 86592339089a..d71c07fb325c 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -1964,7 +1964,9 @@ struct napi_gro_cb {
 	u8	is_ipv6:1;
 
 	/* Number of gro_receive callbacks this packet already went through */
+#ifndef __GENKSYMS__
 	u8 recursion_counter:4;
+#endif
 
 	/* 3 bit hole */
 
-- 
2.10.0

